README.md

```diff
@@ -205,20 +228,28 @@ If you managed to process all the frames, just run the following command from th
 ------------
 
     ├── LICENSE
-    ├── README.md 
+    ├── README.md
     |         <- The top-level README for developers using this project.
-    ├── docs               <- documents
+    ├── annotations        <- annotations in json
+    │   
+    ├── classifier        <- implementation of CNN for litter classification
+    │
+    ├── detr              <- implementation of DETR for litter detection
+    │
+    ├── efficientdet      <- implementation of EfficientDet for litter detection
+    │
+    ├── fastrcnn          <- implementation of FastRCNN for litter segmentation
     │
-    ├── logs          	    <- Tracking experiments e.g. Tensorboard
-    |
-    ├── models             <- Trained and serialized models, model predictions, or model summaries
+    ├── maskrcnn          <- implementation of MaskRCNN for litter segmentation
     │
-    ├── notebooks          <- Jupyter notebooks. 
+    ├── notebooks          <- jupyter notebooks.
+    │   
+    ├── utils              <- source code with useful functions
     │
     ├── requirements.txt   <- The requirements file for reproducing the analysis environment, e.g.
```

Comment:
we need to change requirements list, becouse this one is only for splitting dataset